content/architecture/application/altinn-platform/receipt/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Application architecture receipt component - Altinn Platform
+linktitle: Receipt
+description: Description of receipt component
+tags: [architecture, solution, receipt]
+weight: 100
+---
+
+The profile component is an .NET Core MVC Application exposing a react receipt application, and an internal api for getting the current user.
+
+The receipt frontend view is available at `http://at21.platform.altinn.cloud/receipt/{instanceOwnerId}/{instanceId}`,
+example [https://platform.at21.altinn.cloud/receipt/50002121/a78b3029-009b-4fbc-995d-d8d2abf6f74b](https://platform.at21.altinn.cloud/receipt/50002121/a78b3029-009b-4fbc-995d-d8d2abf6f74b)
```

Comment:
this link might become outdated if we ever clean cosmos for test data or old apps and unavailable once authorization is switched on.